tests/test_regression_entrypoint.py

```
@@ -0,0 +1,139 @@
+                """\
+                #!/usr/bin/env bash
+                exit 0
+                """
+            ),
+        )
+        self._write_executable(bindir / "kubectl", kubectl_script)
+
+        env = os.environ.copy()
+        env["PATH"] = f"{bindir}:{env.get('PATH', '')}"
+
+        return subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)